mediation-xml-to-json/policy.bal

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import choreo/mediation;
+import ballerina/xmldata;
+import ballerina/mime;
+import ballerina/http;
+
+@mediation:InFlow
+public function xmlToJsonIn(mediation:Context ctx, http:Request req) returns http:Response|false|error|() {
```

Comment:
The function does not return which mentioned in return type. Any reason to maintain same signature?

Reply:
Yes. While this particular function may not return what's mentioned in the return type, in general, the functions annotated with `@mediation:XFlow` can have this return type. For now, for simplicity, we are enforcing these functions to have the same return type signature through the compiler plugin in `choreo/mediation` package.